src/settings-ui/Microsoft.PowerToys.Settings.UI/Views/ImageResizerPage.xaml

```diff
@@ -27,150 +34,118 @@
-                                        PointerOverForeground="{ThemeResource ListViewItemForegroundPointerOver}" 
-                                        PressedBackground="{ThemeResource ListViewItemBackgroundPressed}"
-                                        PlaceholderBackground="{ThemeResource ListViewItemPlaceholderBackground}" 
-                                        PointerOverBackground="{ThemeResource ListViewItemBackgroundPointerOver}" ReorderHintOffset="{ThemeResource ListViewItemReorderHintThemeOffset}" 
-                                        SelectedPressedBackground="{ThemeResource ListViewItemBackgroundSelectedPressed}" 
-                                        SelectionCheckMarkVisualEnabled="{ThemeResource ListViewItemSelectionCheckMarkVisualEnabled}" 
-                                        SelectedForeground="{ThemeResource ListViewItemForegroundSelected}" 
-                                        SelectedPointerOverBackground="{ThemeResource ListViewItemBackgroundSelectedPointerOver}" 
-                                        SelectedBackground="{ThemeResource ListViewItemBackgroundSelected}" 
-                                        VerticalContentAlignment="{TemplateBinding VerticalContentAlignment}"/>
-                                    </ControlTemplate>
-                                </Setter.Value>
-                            </Setter>
-                        </Style>
-
-                    </ListView.ItemContainerStyle>
                     <ListView.ItemTemplate>
-                        <DataTemplate x:Name="SingleLineDataTemplate" x:DataType="models:ImageSize" >
-                            <StackPanel Name="ImageResizer_Configurations" x:Uid="ImageResizer_Configurations" Orientation="Horizontal" Padding="0" Spacing="4">
-                                <TextBox Name="ImageResizer_Name"
-                                     x:Uid="ImageResizer_Name"
-                                     Text="{x:Bind Path=Name, Mode=TwoWay}" 
-                                     Width="90"
-                                     VerticalAlignment="Center"
-                                     Margin="{StaticResource SmallTopMargin}"/>
+                        <DataTemplate x:Name="SingleLineDataTemplate" x:DataType="models:ImageSize">
+                            <Grid Padding="0,12,12,12" Width="420">
+                                    <Grid.ColumnDefinitions>
+                                        <ColumnDefinition Width="40" />
+                                        <ColumnDefinition Width="*" />
+                                        <ColumnDefinition Width="52" />
+                                    </Grid.ColumnDefinitions>
+              
+                        
+                                    <FontIcon Glyph="&#xE1D8;" FontSize="18" Margin="0,8,0,0" VerticalAlignment="Top" Grid.RowSpan="2" Foreground="{StaticResource SystemColorControlAccentBrush}" />
```

Comment:
See #10331

Reply:
Besides the fact that this is intended for window resizing.. `E73F` should be the one